index.html

```diff
@@ -0,0 +1,311 @@
+        </div>
+      </div>
+    </section>
+    <section class="special_section">
+      <div class="section_header">
+        Cite Manubot
+      </div>
+      <div class="row">
+        <div>
+          To reference the Manubot project, you can cite 
+          <i><a href="https://github.com/greenelab/meta-review">Open collaborative writing with Manubot</a></i> (AKA the "Meta Review"),
+          a paper that describes the functionality and benefits of Manubot, as follows:
+        </div>
+      </div>
+      <div class="row">
+        <div class="banner_box">
+          <b>Open collaborative writing with Manubot</b><br />
+          Daniel S. Himmelstein, Vincent Rubinetti, David R. Slochower, Dongbo Hu, Venkat S. Malladi, Casey S. Greene, Anthony Gitter<br />
+          <i>Manubot Preprint</i> (2019)
+          <a href="https://greenelab.github.io/meta-review/">
+            https://greenelab.github.io/meta-review/
+          </a>
+        </div>
+      </div>
+    </section>
+    <section>
+      <div class="section_header">
+        Acknowledgements
+      </div>
+      <div class="row">
+        <b>Funded by</b>
+      </div>
+      <div class="row">
+        <figure>
+          <a href="https://sloan.org/"><img  src="alfred-p-sloan-foundation.png" height="60px" /></a>
+          <figcaption>
+            <a href="https://sloan.org/grant-detail/8501">Grant G-2018-11163</a>
+          </figcaption>
+        </figure>
+        <figure>
+          <a href="https://www.moore.org/"><img  src="gordon-betty-moore-foundation.png" height="60px" /></a>
+          <figcaption>
+            <a href="https://www.moore.org/grant-detail?grantId=GBMF4552">Grant GBMF4552</a>
+          </figcaption>
+        </figure>
+      </div>
+      <div class="row">
+        <b>Led by</b>
+      </div>
+      <div class="row">
+        <figure>
+          <a href="https://github.com/dhimmel"><img class="acknowledgement_image" src="daniel-himmelstein.jpg" height="100px" /></a>
+          <figcaption>
+            <span class="semibold">Daniel Himmelstein</span><br />
+            <a href="http://www.greenelab.com/">Greene Lab</a><br />
+            <a href="https://www.upenn.edu/">Univ. of Pennsylvania</a>
+          </figcaption>
+        </figure>
+        <figure>
+          <a href="https://github.com/agitter"><img class="acknowledgement_image" src="anthony-gitter.jpg" height="100px" /></a>
+          <figcaption>
+            <span class="semibold">Anthony Gitter</span><br />
+            <a href="https://morgridge.org/research/virology/gitter-lab/">Morgridge Institute</a><br />
+            <a href="https://www.biostat.wisc.edu/~gitter/">Univ. of Wisconsin-Madison</a>
+          </figcaption>
+        </figure>
+      </div>
+      <div class="row">
+        <div>
+          As an open source project, Manubot relies on a community of maintainers, contributors, and users.
```

Comment:
If we do decide we want to list some of the main contributors or project leads like @slochower and @vincerubinetti, we could name something like a "steering committee" that indicates their influence in defining what Manubot is.  That wouldn't need to be maintained much as new contributors join.

Reply:
I like the idea of a steering committee.